src/coreclr/interpreter/compiler.h

```diff
@@ -179,8 +180,8 @@ struct InterpVar
     int offset;
     int size;
     // live_start and live_end are used by the offset allocator
-    int liveStart;
-    int liveEnd;
+    struct InterpInst* liveStart;
+    struct InterpInst* liveEnd;
```

Comment:
nit: The `struct` keyword can be skipped, but I'm not very familiar with CoreCLR coding conventions. I'm not using it throughout the code.